internal/scan-manager/core/pom.xml

```diff
@@ -155,6 +155,24 @@
             <artifactId>scan-manager-common-external</artifactId>
             <version>${org.wso2.security.tools.scanmanager.common.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Everyone will use these dependencies for tests. Shall we move this to `dependencyManagement` section of https://github.com/wso2/security-tools/blob/master/internal/scan-manager/pom.xml